pkg/api/shared/status.go

```diff
@@ -67,5 +77,15 @@ func GetJsonPatchData(inputResource resources.InputResource) ([]byte, error) {
 	bytes := buf.Bytes()
 	patch := fmt.Sprintf(`[{"op": "replace", "path": "/status/statuses/%s", "value": %s}]`, ns, string(bytes)) // only replace our status so other reporters are not affected (e.g. blue-green of gloo)
 	data := []byte(patch)
+
+	if len(data) > MaxStatusBytes {
+		if contextutils.GetLogLevel() == zapcore.DebugLevel {
```

Comment:
Should we put an informative statement that the status was too large as the status?

Reply:
not sure what you wanted here?